runtime/util/filecache.c

```diff
@@ -65,11 +65,11 @@ assert_cache_separation(struct J9PortLibrary *portLibrary, J9CachedFileHandle* h
 	PORT_ACCESS_FROM_PORT(portLibrary);
 
 	/* go through all the caches to get their ranges */
-	for (i = 0; i < CACHE_BUFFER_NUM; i++) {
+	for (i = 0; i < J9FILECACHE_BUFFER_NUM; i++) {
 		base = handle->cacheTable[i].cacheBase;
-		for (j = i+1; j < CACHE_BUFFER_NUM; j++) {
+		for (j = i+1; j < J9FILECACHE_BUFFER_NUM; j++) {
 			compare_to_base = handle->cacheTable[j].cacheBase;
-			if ( (base < compare_to_base) && ((base + CACHE_BUFFER_SIZE -1) >= compare_to_base) ) {
+			if ( (base < compare_to_base) && ((base + J9FILECACHE_BUFFER_SIZE -1) >= compare_to_base) ) {
```

Comment:
While you're here, can you add the missing spaces that should be both sides of the binary operator `-`?

Reply:
Done here and other places as well.